app/controllers/decidim/votings/admin/votings_controller.rb

```diff
@@ -7,13 +7,13 @@ class VotingsController < Admin::ApplicationController
         helper Decidim::Votings::Admin::VotingsHelper
         helper Decidim::Votings::VotingsHelper
 
-        before_action :init_form_from_params, only: [:create, :update]
-
         def new
           @form = voting_form.instance
         end
 
         def create
+          @form = voting_form.from_params(params)
```

Comment:
I don't completely understand this code (not the change, the existing code)

Can't we say just `VotingForm.from_params(params)`?
What does `form(VotingForm)` does?

Reply:
`form()` is a decidim helper that adds context to the form (current user for example). I don't think we need it right now so I can try to spare it!

Reply:
ok, do what you consider better then :+1:

Reply:
I'll try `VotingForm.from_params(params)` as suggested. It's simpler :+1:

Reply:
Ha, so it turns out adding context is actually necessary (our tests broken when removing it). I think it's because the translated DB columns require access to the current organization in order to validate presence of the default locale.

Something to keep in mind when working with translatable attributes!